### DISCUSSION TITLE
Pushed a bug fix for Linux compilation

### DISCUSSION THREAD

**Discussion by bjdebus** [sandialabs/UQTk#17] [Announcements]
Hi everyone, if you pulled UQTk 3.1.5 recently and tried to compile on linux, you most likely ran into a compilation error. I just pushed a fix that should fix things.

Let us know if you continue to have issues!
Bert

**Comment by evasinha-pnnl** [comment]
@bjdebus I think there is a small bug in `UQTk/PyUQTk/utils/func.py` that gives error `pce_eval: command not found`.

To fix this bug, I added the following at line 47:
`uqtk_bin = os.environ['UQTK_INS'] + "/bin/"`

and modified line 194 to:
`cmd=uqtk_bin + 'pce_eval -x"PC_mi" -f"pccf.dat" -s'+pctype+' -r"mi.dat" > pceval.log'`

and line 221 to:
`cmd=uqtk_bin + 'pce_eval -x"PC" -o' + str(ord) + ' -f"sppc.dat" -s'+pctype+' > pceval.log'`

**Reply by bjdebus** [reply]
Thanks @evasinha for pointing this out. I will fix this in the source code.
Warmly,
Bert
